### ISSUE TITLE
IB Connection status

### ISSUE THREAD

**Issue by vinceyap88** [rterp/SumZeroTrading#27]
There are no status return for InteractiveBrokersClientInterface.connect() method. I have noticed that sometime it will fail to connect the IB. In this case, it is good to have this connect() method to return the status.

**Comment by rterp** [comment]
Agreed, the connect method could either return a true/false boolean status, or the method could throw an exception if it was unsuccessful in connecting.
